src/bindings/text/combinediff.rs

```diff
@@ -0,0 +1,45 @@
+use rlua::Lua;
+
+use std::{io, fs};
+use patch_rs::{Patch, PatchProcessor};
+
+#[cfg(target_os = "windows")]
+const NULL_SOURCE: &str = "nul";
+#[cfg(target_os = "linux")]
+const NULL_SOURCE: &str = "/dev/null";
```

Comment:
I think the only problem with using the defined `NULL_SOURCE` is that they are platform dependent and trying to wrap my head around how they will be used if `nul` is used on linux and vice versa. I havent checked the combinediff code though but some clarification would be helpful on how either `patch_1` or `patch_2` will equal `NULL_SOURCE`

Reply:
I am not moving the `NULL_SOURCE` away from `torchbear` yet, because `patchutils` do not actually work with files, but with parsed data or vectors of strings instead.
Probably, the `torchbear` maintainers would like to have more control over this constant in the future.
Needs @mitchtbaum 's clarification.

Reply:
Concerning `nul`, on Linux it is a valid file, but on Windows this name is invalid.
Probably, `torchbear` users would like to use a platform-independent string to specify an empty file. Perhaps, the string must contain characters, which are invalid for file names on all platforms.